source/common/common/backoff_strategy.cc

```diff
@@ -15,9 +15,10 @@ uint64_t JitteredBackOffStrategy::nextBackOffMs() {
   if (base_backoff <= max_interval_) {
     current_retry_++;
   }
+  ASSERT(base_backoff > 0);
   return std::min(random_.random() % base_backoff, max_interval_);
 }
 
 void JitteredBackOffStrategy::reset() { current_retry_ = 1; }
 
-} // namespace Envoy
\ No newline at end of file
+} // namespace Envoy
```

Comment:
an incidental change picked up by my vim settings, but leaving it in since we probably want all files to end with newline.

Reply:
It would be nice to make the formatter deal with this...